doc/classes/TileData.xml

```diff
@@ -123,6 +137,22 @@
 				Sets the polygons count for TileSet physics layer with index [code]layer_id[/code].
 			</description>
 		</method>
+		<method name="set_constant_angular_velocity">
+			<return type="void" />
+			<argument index="0" name="layer_id" type="int" />
+			<argument index="1" name="velocity" type="float" />
+			<description>
+				Sets the constant angular velocity. This angular velocity is applied to objects colliding with this tile.
```

Comment:
Just to be perfectly clear, this could explicitly say that it doesn't rotate the tile. Something like:
```suggestion
				Sets the constant angular velocity. This does not rotate the tile. This angular velocity is applied to objects colliding with this tile.
```